test/library/packages/UnitTest/Launcher.chpl

```diff
@@ -58,6 +58,23 @@ module Launcher {
         if files.size == 0 && dirs.size == 0 {
           dirs.push_back(".");
         }
+        // set comm if found empty
+        if comm == "" then comm = "none";
```

Comment:
I just realized there is an error in our strategy here -- `CHPL_COMM` default value may depend on several factors if `CHPL_COMM` is not explicitly set.

I think we're going to actually need to have our launcher call `$CHPL_HOME/util/printchplenv` or `python $CHPL_HOME/util/chplenv/chpl_comm.py` directly to determine the `CHPL_COMM` value.

This could be wrapped in a clean interface like `proc getRuntimeComm();`, which should eliminate the need for a lot of conditional logic here.

Reply:
okay sir.
I have made the changes.